setuptools/config.py

```diff
@@ -308,11 +308,8 @@ def _parse_file(cls, value):
         """Represents value as a string, allowing including text
         from nearest files using `file:` directive.
 
-        Directive is sandboxed and won't reach anything outside
-        directory with setup.py.
```

Comment:
This change removes the intentional constraint to disallow loading from outside the project directory. This change will affect any config option that uses `_parse_file`. If we wish to change the behavior here to be more lenient, we should at least enumerate the directives that might use this behavior. Is it only `long_description` or are there others?